compile.sh

```diff
@@ -76,12 +76,46 @@ OPTS+=" -Dhadoopversion=$HADOOP_VERSION"
 OPTS+=" -Dsparkversion=$SPARK_VERSION"
 OPTS+=" $PLAY_OPTS"
 
-set -x
-trap "exit" SIGINT SIGTERM
 
 project_root=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 cd ${project_root}
 
+cd ${project_root}
+
+
+#if npm is installed, install bower,ember-cli and other components for new UI
```

Comment:
I'm writing the complete documentation for the new UI. Once I have it completed, I'll update the documentation.  The compilation of new UI is dependent on the installation of npm so it is optional right now. Also, even if the compilation for new UI fails with npm, the older UI will still be present.